packages/ui-core/src/components/bulletlist.component.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type BulletPoint = {
+  color: string;
+  value: string;
+};
+
+const BulletPoint = (props: { color: string }) => {
+  const Bullet = styled.div`
+    width: 8px;
+    height: 8px;
+    margin-right: 6px;
+    border-radius: 50%;
+    background-color: ${props.color};
+  `;
+  return <Bullet />;
+};
+
+const BulletList = (props: { list: BulletPoint[] }) => {
+  const StyledBulletList = styled.ul`
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  `;
+  const StyledBulletItem = styled.li`
```

Comment:
This component definition cannot be placed in function body as each re-render will create a new instance of component and hoist to the top a new `css` definition. Please move them out of function.